super-intel.c

```diff
@@ -4357,7 +4357,7 @@ static void end_migration(struct imsm_dev *dev, struct intel_super *super,
 		map_state = imsm_check_degraded(super, dev, failed, MAP_0);
 	}
 
-	dev->vol.migr_state = 0;
+	dev->vol.migr_state |= 0;
```

Comment:
It was clearing it, now it is definitely not clearing it. I did simple program:
```c
#include <stdio.h>

int main()
{
    int v = 1;
    printf("%d\n", v);
    
    v |= 0;
    
    printf("%d\n", v);

    return 0;
}
```
Result is: 
>1
>1

This is wrong to me.